fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/DataDefinitionUtil.java

```diff
@@ -109,6 +110,16 @@ public static String join(Collection<String> things) {
         return things.stream().collect(Collectors.joining(","));
     }
 
+    /**
+     * Join the ordered column definitions
+     * @param things
+     * @return
+     */
+    public static String joinOrderedColumnDefs(Collection<OrderedColumnDef> things) {
+        return things.stream().map(c -> c.toString()).collect(Collectors.joining(","));
```

Comment:
Is this always going to be an ArrayList or ordered list underneath?

Reply:
Probably. Why?

Reply:
I read the OrderedColumDef afterwards (I see how it orders now)